data/kramdown-rfc2629.erb

```diff
@@ -12,14 +12,28 @@
 <% end -%>
 ]>
 
-<% ps.arr("pi", false) do |pi, val| -%>
+<%
+  rfcattrs = ps.attrs("ipr", "docName=docname", "category=cat",
+                      "consensus", "submissionType=submissiontype", "xml:lang=lang",
+                      "number", "obsoletes", "updates", "seriesNo=seriesno")
+  if $options.v3
+    ps.arr("pi", false) do |pi, val|
+      keys = ["tocdepth", "toc", "sortrefs", "symrefs"]
+      if keys.index(pi) != nil && !ps.has(pi)
+        ps.add(pi, {true => "true", false => "false", nil => "true"}[val] || val)
+      end
+    end
+    if piattrs = ps.attrs("tocDepth=tocdepth", "tocInclude=toc",
```

Comment:
Wouldn't it be easier to come from pi.attrs?

Reply:
Ahh, I see where you are going with this.  I was looking to promote these attributes to the top level, but you are probably right that keeping these instructions under `pi` is the right answer.  I'll look into it.  The need to remap values is somewhat challenging though.